### ISSUE TITLE
Error with graph() with networkx≥2.0

### ISSUE THREAD

**Issue by j6k4m8** [iscoe/pytri#49]
When drawing edges with `pytri#graph`:

```
TypeError: Cannot use 'in' operator to search for 'pos' in undefined
    at GraphLayer._getNodePosition
```

With `networkx>=2.0` (which is currently what we auto-install when you pip install pytri...)

**Comment by j6k4m8** [comment]
also: `add_node("id", {"foo": "bar"})` → `add_node("id", foo="bar")`

**Comment by j6k4m8** [comment]
@hpcowley fixed in #71, right?

**Comment by hpcowley** [comment]
Confirmed. Fixed in #71
